### PR TITLE
cleanup /etc/fstab handling for Intel and ARP

### PR DESCRIPTION
**Pull request by martin-ejdestig**
Work in progress. Will hold off with this until after 4.0 release.

### CODE REVIEW

**Comment by martin-ejdestig** [comment]
Should perhaps consider removing the hardcoded UUID:s completely. As far as I can tell they are only used to specify PARTUUID in bootloader for root. But labels should work there as well, I think.

File: conf/variant/arp-qtauto/bblayers.conf.sample [hunk 1]
```diff
@@ -7,7 +7,6 @@ require conf/variant/common/bblayers-qtauto.conf
 
 BBLAYERS += "\
     ${BSPDIR}/sources/meta-pelux/meta-intel-extras  \
-    ${BSPDIR}/sources/meta-pelux/meta-arp-extras    \
```
Comment (sashko) [inline conf/variant/arp-qtauto/bblayers.conf.sample]: why?

Comment (martin-ejdestig) [reply conf/variant/arp-qtauto/bblayers.conf.sample]: It was only added for the /etc/fstab workaround.

**Comment by martin-ejdestig** [comment]
4.0 has been released and no longer wip, ready for review and merge.

**Pull request merged**